RLTSquare/Unit3/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <tab id="rltsquare" sortOrder="1">
+            <label>RLTSquare</label>
+        </tab>
+        <section id="unit3" sortOrder="10" showInDefault="1" showInStore="1" showInWebsite="1" type="text">
+            <tab>rltsquare</tab>
+            <label>Unit 3</label>
+            <resource>RLTSquare_Unit3::config</resource>
+            <group id="settings" type="text" showInWebsite="1" showInStore="1" showInDefault="1" sortOrder="20">
+                <label>Settings</label>
+                <field id="username" translate="label" type="text" sortOrder="10" showInDefault="1" showInWebsite="1"
+                       showInStore="1">
+                    <label>Username</label>
+                    <validate>required-entry validate-data no-whitespace</validate>
+                    <comment>Enter a username.</comment>
+                </field>
+                <field id="password" translate="label" type="obscure" sortOrder="20" showInDefault="1" showInWebsite="1"
```

Comment:
There should be separate field for staging password, separate for production, etc... 

```
<field id="environment" translate="label" type="select" sortOrder="10" showInDefault="1" showInWebsite="1" showInStore="1">
    <label>Enabled</label>
    <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
</field>
<field id="staging_pass" translate="label" type="text" sortOrder="15" showInDefault="1"   showInWebsite="1" showInStore="1">
    <label>Password</label>
    <depends>
        <field id="environment">staging</field>
    </depends>
    <validate>required-entry validate-password no-whitespace</validate>
    <backend_model>Magento\Config\Model\Config\Backend\Encrypted</backend_model>
</field>
<field id="production_pass" translate="label" type="text" sortOrder="15" showInDefault="1"   showInWebsite="1" showInStore="1">
    <label>Password</label>
    <depends>
        <field id="environment">production</field>
    </depends>
    <validate>required-entry validate-password no-whitespace</validate>
    <backend_model>Magento\Config\Model\Config\Backend\Encrypted</backend_model>
</field>
```

Within the helper getPassword, you would still return single pass based of selected environment.